iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -168,7 +171,8 @@ internal sealed class MqttTransportHandler : TransportHandler, IMqttIotHubEventH
             _deviceboundMessageFilter = string.Format(CultureInfo.InvariantCulture, DeviceBoundMessagesTopicFilter, iotHubConnectionString.DeviceId);
             _deviceboundMessagePrefix = string.Format(CultureInfo.InvariantCulture, DeviceBoundMessagesTopicPrefix, iotHubConnectionString.DeviceId);
 
-            _qos = settings.PublishToServerQoS;
+            _qosSendPacketToService = settings.PublishToServerQoS;
+            _qosReceivePacketFromService = settings.ReceivingQoS;
```

Comment:
What change in behavior might this have for customers?

Reply:
Not much, really. 
For customers who don't set this value in settings, the default for both is 1 - they wouldn't see any difference. 
For the ones that do set it explicitly to something else, they still wouldn't see much of a difference either since IoT hub mqtt implementation isn't a true broker. For us, it means that receive C2D messaging is the only call that really looks at the QoS. For the other calls, the service doesn't call out how to configure it. What that means is, even if we subscribe to twin property updates with a QoS of 1, service might still be publishing the packets with QoS of 0. They don't state the contract as a part of the hub-mqtt implementation.
https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-mqtt-support